src/playwright/matchers/to-be-in-the-custom-element-registry.ts

```diff
@@ -4,15 +4,19 @@ import { type Locator, expect } from '@playwright/test';
  * Asserts that an element is in the custom element registry.
  */
 export default expect.extend({
-  async toBeDefined(locator: Locator, name: string) {
-    const isDefined = await locator.evaluate((node, name) => {
+  async toBeInTheCustomElementRegistry(locator: Locator, name: string) {
+    const isInTheRegistry = await locator.evaluate((node, name) => {
```

Comment:
Splitting hairs, as always. Contemplated leaving `isDefined` - I waffle on naming. Went ahead and updated it to be more explicit. Although it's really checking that a) it's in the registry with that name and b) it's the same as the current node, but 🤷

Reply:
It's doing those two things. But the result is whether or not the thing is in the registry. Don't overthink it! 😆